test/spy-test.js

```diff
@@ -284,6 +284,7 @@ describe("spy", function () {
         var originalError = global.Error;
         try {
             assert(createSpy(global, "Error"));
```

Comment:
Nice fixes, but we prefer that you keep related changes in separate commits. These code changes have little to do with the commit message, so no one looking at `git blame` will understand why these changes were made.

Could you split the commit and give a little explanation to what you are fixing with these changes? I can _see_ it for myself, but it would be nice to have it logged too 😺